next.config.tmp.js

```diff
@@ -1,9 +1,43 @@
-module.exports = {
-  env: {
-    API_BASE_URL: 'http://localhost:3001',
+const { PHASE_PRODUCTION_BUILD } =
+  process.env.NODE_ENV === 'development'
+    ? {} // We're never in "production server" phase when in development mode
+    : !process.env.NOW_REGION
+      ? require('next/constants') // Get values from `next` package when building locally
+      : require('next-server/constants') // Get values from `next-server` package when building on now v2
+
+module.exports = (phase, { defaultConfig }) => {
+  // Add loader for markdown files.
+  const webpack = (config) => {
+    config.module.rules.push(
+      {
+        test: /\.md$/,
+        use: 'raw-loader'
+      }
+    )
+    return config
+  }
+  let env = {
+    API_BASE_URL: 'http://localhost:4567',
+    API_KEY: 'your_api_key',
     AUTH0_DOMAIN: 'YOUR_DOMAIN.auth0.com',
     AUTH0_CLIENT_ID: 'AUTH0_CLIENT_ID',
     REDIRECT_URI: 'http://localhost:3000/',
     POST_LOGOUT_REDIRECT_URI: 'http://localhost:3000/'
```

Comment:
Could we use something like `let env = require('env.yml')` to be consistent with other projects?

Reply:
I start to feel strongly about moving this config stuff into a yml file. @evansiroky what do you think?

Reply:
We have [this](https://github.com/ibi-group/datatools-ui/blob/dev/__tests__/test-utils/utils.js#L97) in datatools-ui and configurations for middleware.

Reply:
I don't think this should be a dealbreaker for this PR. This is the documented way to handle config in next.js, so I think perhaps we should just leave it as is for now and revisit if need be.